chipy_org/apps/meetings/models.py

```diff
@@ -236,9 +271,25 @@ def __str__(self):
 
 class RSVP(CommonModel):
 
-    RSVP_CHOICES = (
-        ("Y", "Yes"),
-        ("N", "No"),
+    YES = "Y"
+    NO = "N"
+    RSVP_RESPONSE = ((YES, "Yes"), (NO, "No"))
+
+    ACCEPTED = "A"
+    DECLINED = "D"
+    WAIT_LISTED = "W"
+
+    RSVP_STATUS = (
+        (ACCEPTED, "Accepted"),
+        (DECLINED, "Declined"),
+        (WAIT_LISTED, "Wait Listed"),
+    )
```

Comment:
The RSVP statuses seem more business logic than finite state machine (which is what you want, I think). Much more common to go with a pattern like: 

> pending - no decision made
 confirmed - rsvp request can be fulfilled
 rejected - rsvp request cannot be fulfilled
 waitlisted - rsvp request could not be fulfilled on prior attempt, may be fulfilled in future

Pending might be YAGNI, but if you move to a worker RSVP model (single worker to avoid race conditions) then you'll be ready out of the gate.

Reply:
Done.